modules/ffmpeg/producer/ffmpeg_producer.cpp

```diff
@@ -637,7 +652,8 @@ void describe_producer(core::help_sink& sink, const core::help_repository& repo)
 	sink.example(L">> PLAY 1-10 v4l2:///dev/video0", L"to use a web camera as video input on Linux.");
 	sink.para()->text(L"The FFmpeg producer also supports changing some of the settings via ")->code(L"CALL")->text(L":");
 	sink.example(L">> CALL 1-10 LOOP 1");
-	sink.example(L">> CALL 1-10 START 10");
+	sink.example(L">> CALL 1-10 IN 10");
+	sink.example(L">> CALL 1-10 OUT 60");
 	sink.example(L">> CALL 1-10 LENGTH 50");
 	sink.example(L">> CALL 1-10 SEEK 30");
 	core::describe_framerate_producer(sink);
```

Comment:
After `framerate_producer` has appended its help you could have some combination of `framerate speed` and `framestep` examples. You can check with `HELP producer ffmpeg producer` to see if they combine well.